examples/modules/plus-number/CMakeLists.txt

```diff
@@ -19,38 +19,28 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN)
 
-# Prefix an suffix of Library name for different OS.
-if (WIN32)
-    set(LIBRARY_PREFIX "")
-    set(LIBRARY_SUFFIX ".lib")
-elseif (APPLE)
-    set(LIBRARY_PREFIX "lib")
-    set(LIBRARY_SUFFIX ".dylib")
-else()
-    set(LIBRARY_PREFIX "lib")
-    set(LIBRARY_SUFFIX ".so")
-endif()
+execute_process(COMMAND node -e "require('napajs/build').paths.inc" RESULT_VARIABLE ERR OUTPUT_VARIABLE NAPAJS_INC)
+if(ERR)
+    message(FATAL_ERROR "Failed to get NAPAJS_INC")
```

Comment:
>NAPAJS_INC [](start = 39, length = 10)

This is an internal MACRO, please correct the error message. Something like:
"Failed to resolve napa include dir"